src/components/styledComponents/index.tsx

```diff
@@ -0,0 +1,44 @@
+import styled from 'styled-components';
+import { Col, Form } from 'antd';
+import { FormItemProps } from 'antd/es/form';
+import { BLACK, LIGHT_GREY } from '../../utils/colors';
+
+export const ContentContainer = styled.div`
+  display: block;
+  padding: 3vw;
+  width: 70vw;
+  margin: auto;
+`;
+
+export const CenterDiv = styled.div`
+  margin: auto;
+`;
+
+export const InputContainer = styled(Col)`
+  height: 60vh;
+  width: 60vw;
+  padding: 30px 120px 0 50px;
+  background: ${LIGHT_GREY};
+  box-shadow: 2px 3px 6px ${BLACK}25;
+  border-radius: 6px;
+  overflow: auto;
+`;
+
+export const TabletPageContainer = styled.div`
+  margin: auto;
+  width: 70vw;
+`;
+
+export const FormRow = styled.div`
+  width: 100%;
+`;
+
+export const Gap = styled.div`
```

Comment:
Either way works but I preferred this over having a LeftHalfItem and RightHalfItem that are identical but have different margins. I also thought this was slightly more flexible, since there might be several places where you have 2+ identical items that need a gap between them.